include/mbedtls/ssl.h

```diff
@@ -125,6 +125,11 @@
 #define MBEDTLS_ERR_SSL_HELLO_VERIFY_REQUIRED             -0x6A80  /**< DTLS client must retry for hello verification */
 #define MBEDTLS_ERR_SSL_BUFFER_TOO_SMALL                  -0x6A00  /**< A buffer is too small to receive or write a message */
 #define MBEDTLS_ERR_SSL_NO_USABLE_CIPHERSUITE             -0x6980  /**< None of the common ciphersuites is usable (eg, no suitable certificate, see debug messages). */
+/*
+ * MBEDTLS_ERR_SSL_WANT_READ and MBEDTLS_SSL_WANT_WRITE are dismissable errors,
```

Comment:
I missed it at first, but `check-names.sh` point out that `MBEDTLS_SSL_WANT_WRITE` is likely a typo - after staring at it for a good 30 seconds I realised it's indeed missing the `ERR_` part.

Reply:
Thanks for noticing, I was staring it at too without noticing :)